build.sbt

```diff
@@ -88,65 +32,43 @@ val logbackClassicV = "1.2.10"
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
-lazy val log4cats = project
-  .in(file("."))
-  .aggregate(
-    coreJVM,
-    coreJS,
-    testingJVM,
-    testingJS,
-    noopJVM,
-    noopJS,
-    slf4j,
-    docs
-  )
-  .enablePlugins(NoPublishPlugin)
-  .settings(commonSettings, releaseSettings)
+lazy val root = tlCrossRootProject.aggregate(core, testing, noop, slf4j, docs)
 
 lazy val docs = project
-  .settings(commonSettings, micrositeSettings)
-  .enablePlugins(MicrositesPlugin)
-  .enablePlugins(MdocPlugin)
-  .enablePlugins(NoPublishPlugin)
-  .settings(mdocIn := sourceDirectory.value / "main" / "mdoc")
+  .in(file("site"))
+  .enablePlugins(TypelevelSitePlugin)
   .dependsOn(slf4j)
 
 lazy val core = crossProject(JSPlatform, JVMPlatform)
-  .settings(commonSettings, releaseSettings)
+  .settings(commonSettings)
   .settings(
     name := "log4cats-core",
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % catsV
     )
   )
-lazy val coreJVM = core.jvm
-lazy val coreJS = core.js
 
 lazy val testing = crossProject(JSPlatform, JVMPlatform)
-  .settings(commonSettings, releaseSettings)
+  .settings(commonSettings)
   .dependsOn(core)
   .settings(
     name := "log4cats-testing",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-effect"     % catsEffectV,
-      "ch.qos.logback"  % "logback-classic" % logbackClassicV % Test
+      "org.typelevel" %%% "cats-effect" % catsEffectV,
+      "ch.qos.logback"                  % "logback-classic" % logbackClassicV % Test
     )
   )
-lazy val testingJVM = testing.jvm
-lazy val testingJS = testing.js
 
 lazy val noop = crossProject(JSPlatform, JVMPlatform)
-  .settings(commonSettings, releaseSettings)
+  .settings(commonSettings)
   .dependsOn(core)
   .settings(
     name := "log4cats-noop"
   )
-lazy val noopJVM = noop.jvm
-lazy val noopJS = noop.js
 
 lazy val slf4j = project
-  .settings(commonSettings, releaseSettings)
-  .dependsOn(coreJVM)
+  .settings(commonSettings)
+  .dependsOn(core.js)
```

Comment:
Not a blocker at all, just curious, why did you change the `coreJVM` to `core.js` here?

Reply:
See my comment https://github.com/typelevel/log4cats/pull/592/files#r794407004. It's a style of code I usually see in legacy builds.